core/audits/seo/crawlable-anchors.js

```diff
@@ -61,7 +64,7 @@ class CrawlableAnchors extends Audit {
       if (rawHref.startsWith('file:')) return true;
       if (name.length > 0) return;
 
-      if (rawHref === '') return true;
+      if (rawHref === '' && href === '') return true;
```

Comment:
I think we should just check `href`, I'm not sure this exact condition is possible because `rawHref === ''` means that `href` will be the current URL.

```suggestion
      if (href === '') return true;
```